tests/ci/integration/run_mysql_integration.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -exu
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+source tests/ci/common_posix_setup.sh
+
+MYSQL_VERSION_TAG="mysql-8.0.33"
+# This directory is specific to the docker image used. Use -DDOWNLOAD_BOOST=1 -DWITH_BOOST=<directory>
+# with mySQL to download a compatible boost version locally.
+BOOST_INSTALL_FOLDER=/home/dependencies/boost
+
+# Set up environment.
+
+# ROOT
+#  |
+#  - AWS_LC_DIR
+#    |
+#    - aws-lc
+#  |
+#  - SCRATCH_FOLDER
+#    |
+#    - mysql
+#    - AWS_LC_BUILD_FOLDER
+#    - AWS_LC_INSTALL_FOLDER
+#    - MYSQL_BUILD_FOLDER
+
+# Assumes script is executed from the root of aws-lc directory
+AWS_LC_DIR=$(pwd)
+cd ../
+ROOT=$(pwd)
+
+SCRATCH_FOLDER=${ROOT}/"MYSQL_BUILD_ROOT"
+MYSQL_SRC_FOLDER="${SCRATCH_FOLDER}/mysql-server"
+MYSQL_BUILD_FOLDER="${SCRATCH_FOLDER}/server/mysql-aws-lc"
+AWS_LC_BUILD_FOLDER="${SCRATCH_FOLDER}/aws-lc-build"
+AWS_LC_INSTALL_FOLDER="${MYSQL_SRC_FOLDER}/aws-lc-install"
+
+mkdir -p ${SCRATCH_FOLDER}
+rm -rf ${SCRATCH_FOLDER}/*
+cd ${SCRATCH_FOLDER}
+
+function aws_lc_build() {
+  ${CMAKE_COMMAND} "${AWS_LC_DIR}" -GNinja "-B${AWS_LC_BUILD_FOLDER}" "-DCMAKE_INSTALL_PREFIX=${AWS_LC_INSTALL_FOLDER}" "$@"
+  ninja -C "${AWS_LC_BUILD_FOLDER}" install
+  ls -R "${AWS_LC_INSTALL_FOLDER}"
+  rm -rf "${AWS_LC_BUILD_FOLDER:?}"/*
+}
+
+function mysql_build() {
+  cmake ${MYSQL_SRC_FOLDER} -GNinja -DENABLED_PROFILING=OFF -DWITH_NDB_JAVA=OFF  -DWITH_BOOST=${BOOST_INSTALL_FOLDER} -DWITH_SSL=${AWS_LC_INSTALL_FOLDER} "-B${MYSQL_BUILD_FOLDER}"
+  ninja -C ${MYSQL_BUILD_FOLDER}
+  ls -R ${MYSQL_BUILD_FOLDER}
+}
+
+function mysql_run_tests() {
+  pushd ${MYSQL_BUILD_FOLDER}
+  ninja test
+  popd
+}
+
+# Get latest MySQL version. MySQL often updates with large changes depending on OpenSSL all at once, so we pin to a specific version.
+git clone https://github.com/mysql/mysql-server.git ${MYSQL_SRC_FOLDER} -b ${MYSQL_VERSION_TAG} --depth 1
```

Comment:
We don't want our CI to fail unpredictably when there's a new MySql releases, but we also don't want to forget to update the tag periodically.

It would be nice if we could generate a metric/alarm when the tag we're using doesn't match the latest release.
```
LATEST_MYSQL_VERSION_TAG=mysql-`curl https://api.github.com/repos/mysql/mysql-server/tags | jq '.[].name' |grep '\-8.0' |sed -e 's/"mysql-cluster-\(.*\)"/\1/' |sort | tail -n 1`
```

Reply:
Great point, I'll see if there's something we can do to cut a ticket to us and remind us to do so. Thanks for the Tag code :).

Reply:
I updated the script to print us a warning. I briefly looked into it again, but Codebuild doesn't seem to have a "log tracking" mechanism in place. We had some similar discussion about creating an alarm here: https://github.com/aws/aws-lc/pull/1042#discussion_r1214951789. I'll update the SIM to make it more general.
I think we could consider creating a Github Action to poke us instead

Reply:
In this it's actually a bit easier, we can just call cloudwatch put metric in this bash script, what's a little harder from 1042 is it's a string output from our test.